packages/playwright-crawler/package.json

```diff
@@ -64,7 +64,7 @@
         "ow": "^0.28.1"
     },
     "peerDependencies": {
-        "playwright": "^1.21.1"
+        "playwright": ">= 1.21.x <= 1.27.x"
```

Comment:
other places to adjust:
<img width="267" alt="image" src="https://user-images.githubusercontent.com/615580/204294536-76f29dca-d747-455a-b28b-534df05a8b57.png">